lib/generators/templates/models/rate.rb

```diff
@@ -0,0 +1,3 @@
+class Rate < BookingsyncPortal::Rate
+  synced delegate_attributes: [:name], remove: true
```

Comment:
We have a `Rate` endpoint in API v3 http://developers.bookingsync.com/reference/endpoints/rates/